src/autoscaler/pruner/config/config_test.go

```diff
@@ -380,8 +372,8 @@ lock:
 				conf.Lock.ConsulClusterConfig = ""
 			})
 
-			It("should error", func() {
-				Expect(err).To(MatchError(MatchRegexp("Configuration error: Consul Cluster Config is empty")))
+			It("should validate successfully", func() {
+				Expect(err).NotTo(HaveOccurred())
```

Comment:
@kanekoh This test validates the config without consul-cluster-config section.partial config test is basically testing whether default values for different properties are correctly applied with the partial (mandatory) properties or not. IMHO we should keep this.